internal/venice-common/src/test/java/com/linkedin/venice/kafka/validation/TestProducerTracker.java

```diff
@@ -209,7 +209,7 @@ public void testSequenceNumber() {
     ConsumerRecord<KafkaKey, KafkaMessageEnvelope> fourthConsumerRecord = new ConsumerRecord<>(
         topic,
         partitionId,
-        offset++,
+        offset,
```

Comment:
So we dont need to increment the offset ?

Reply:
Well, at this line, incrementing the offset has not effect at all.
The logic of the test case is like
```
// do something A here
    ConsumerRecord<KafkaKey, KafkaMessageEnvelope> thirdConsumerRecord = new ConsumerRecord<>(
        topic,
        partitionId,
        offset++, // increment it once 
        System.currentTimeMillis() + 1000,
        TimestampType.NO_TIMESTAMP_TYPE,
        ConsumerRecord.NULL_CHECKSUM,
        ConsumerRecord.NULL_SIZE,
        ConsumerRecord.NULL_SIZE,
        thirdMessageKey,
        thirdMessage);

// do something B here

    ConsumerRecord<KafkaKey, KafkaMessageEnvelope> thirdConsumerRecord = new ConsumerRecord<>(
        topic,
        partitionId,
        offset, // no need to increment because this is the end of test. It's a redundant operation 
        System.currentTimeMillis() + 1000,
        TimestampType.NO_TIMESTAMP_TYPE,
        ConsumerRecord.NULL_CHECKSUM,
        ConsumerRecord.NULL_SIZE,
        ConsumerRecord.NULL_SIZE,
        thirdMessageKey,
        thirdMessage);

// done
```

Reply:
the test is not using it,  but trying to simulate the increasing offset from producer.